Libraries/src/Amazon.Lambda.KafkaEvents/README.md

```diff
@@ -0,0 +1,38 @@
+# Amazon.Lambda.KafkaEvents
+
+This package contains classes that can be used as input types for Lambda functions that process Apache Kafka events.
+
+# Serialization
+
+If you are using this package with Amazon Lambda but are not also using `Amazon.Lambda.Serialization.Json`, be aware that one property requires custom serialization.
+
+This is a `MemoryStream` object that should be populated by converting the JSON string from base64 to an array of bytes, then constructing a `MemoryStream` object from these bytes. Here is a code sample showing this deserialization logic.
+```csharp
+string dataBase64 = GetJsonString();
+byte[] dataBytes = Convert.FromBase64String(dataBase64);
+MemoryStream stream = new MemoryStream(dataBytes);
+```
```

Comment:
I didn't fully understand purpose of this. Where is `MemoryStream` object used? Did you mean Value inside the record?

Reply:
I think this section should be removed because there is nothing special for serialization here.

Reply:
Fixed.